src/test/transaction_tests.cpp

```diff
@@ -342,11 +342,26 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     string reason;
     BOOST_CHECK(IsStandardTx(t, reason));
 
-    t.vout[0].nValue = 501; // dust
+    // Check dust with default relay fee:
+    CAmount nDustThreshold = 182 * minRelayTxFee.GetFeePerK()/1000 * 3 ;
```

Comment:
There is an unneeded space at the end.

Other than that, for this line you may consider using:
```cpp
CAmount nDustThreshold = t.vout[0].GetDustThreshold(minRelayTxFee);
```

It should have a similar effect. Since hardcoded values are used (such as `546`), it might be good to specify the "default relay fee" in the comment one line above.